docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,274 @@
+# Deploying a Mesos Cluster with Calico on CentOS7
+
+In these instructions, we will install Mesos and Calico Networking on baremetal CentOS/Red Hat Enterprise Linux 7 from rpm packages. These packages can be built from the Mesosphere [net-modules repo](https://github.com/mesosphere/net-modules) and will install the following processes:
```

Comment:
Remove "Red Hat Enterprise Linux" from here. The two are actually different, and this install is for CentOS7.